com_redhat_kdump/service/kdump.py

```diff
@@ -85,6 +86,18 @@ def reserved_memory(self):
 
     @reserved_memory.setter
     def reserved_memory(self, value):
+        if value != "auto":
```

Comment:
https://github.com/rhinstaller/kdump-anaconda-addon/blob/master/com_redhat_kdump/service/kickstart.py#L96-L104 already makes sure` value` is a int. So there is no need to remove the suffix here.

Reply:
https://github.com/rhinstaller/kdump-anaconda-addon/blob/master/com_redhat_kdump/gui/spokes/kdump.py#L143-L145, GUI to set 'auto' or 'XXM' as a string to set KdumpService::reserved_memory.When it is 'XXM'  so need to check 'suffix'.
https://github.com/rhinstaller/kdump-anaconda-addon/blob/master/com_redhat_kdump/gui/spokes/kdump.py#L101-L108, if the GUI get the value with no 'M' and set with no string i think can simplify.

Reply:
So  the value set in GUI will be passed to com_redhat_kdump/service/kdump.py, am I correct? Even if that's the case, GUI also makes sure it's a int value,
```py
    def refresh(self):
        # If a reserve amount is requested, set it in the spin button
        # Strip the trailing 'M'
        reserveMB = self._proxy.ReservedMemory
        if reserveMB != "auto":
            if reserveMB and reserveMB[-1] == 'M':
                reserveMB = reserveMB[:-1]
            if reserveMB:
                self._toBeReservedSpin.set_value(int(reserveMB))

```